apps/wallet-mobile/src/features/RampOnOff/common/ShowBuyBanner/BuyBannerBig.tsx

```diff
@@ -31,15 +31,15 @@ export const BuyBannerBig = () => {
       <LinearGradient style={styles.gradient} start={{x: 1, y: 1}} end={{x: 1, y: 1}} colors={colors.gradientColor}>
         <BuyBannerIllustration width={bannerWidth} height={bannerHeight} />
 
-        <Spacer />
+        <Space />
 
         <Text style={styles.label}>{strings.getFirstCrypto}</Text>
 
-        <Spacer height={4} />
+        <Space height="xl" />
```

Comment:
xl = 4 ?

Reply:
Thanks for noticing it! I was changing it so much and forgot about it